dev/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+pg:
+  image: postgres:9.3
+  
+possum:
+  image: possum
+  command: server -a cucumber
+  environment:
+    DATABASE_URL: postgres://postgres@pg/postgres
+    POSSUM_DATA_KEY:
+  links:
```

Comment:
i'm pretty sure you don't need links here anymore, now that `docker-compose` services all use the same network by default. You already named the service `pg` up above so it should just work if you:
1. add `version: '2'` at the top of the file
2. nest the different containers under a new `services:` block